### ISSUE TITLE
STweep throws a no solution found exception

### ISSUE THREAD

**Issue by Barteling** [Barteling/STweep#40]
Got a similar exception:

![image](https://user-images.githubusercontent.com/20192767/114671840-d2620000-9d04-11eb-872e-975167302e7e.png)

when attempting to format 

```
FUNCTION_BLOCK POU
VAR
	Parameter : ARRAY[1..3] OF ST_Parameters := [
		(A:=1, B:=1, B:=1),
		(A:=1, B:=1, B:=1), 
		(A:=1, B:=1, B:=1)
	];
END_VAR
```

Using [settings.txt](https://github.com/Barteling/STweep/files/6309047/settings.txt) and STweep v3.4.0

_Originally posted by @Roald87 in https://github.com/Barteling/STweep/issues/37#issuecomment-819304904_

**Comment by Barteling** [comment]
although it's the same message I don't think it related to issue #37 so I created a new issue
I'll check whats going on.

**Comment by Barteling** [comment]
Fixed this in 3.5.0-Beta 
https://www.stweep.com/Beta-releases/
Codesys version is to follow soon
